File: examples/python/tasks/token_classification/token_classification.py
from bytez import Bytez

client = Bytez("YOUR_BYTEZ_KEY_HERE")

model = client.model("dslim/bert-base-NER")

model.load()

input_text = "John Doe is a software engineer at Google."

result = model.run(input_text)

word_objects = result["output"]

for word_object in word_objects:
    # Depending on the model, there may be additional props returned
    print(word_object)

    word = word_object.get("word")
    entity = word_object.get("entity")
    score = word_object.get("score")
    index = word_object.get("index")
    start = word_object.get("start")
    end = word_object.get("end")

    print(
        {
            "word": word,
            "entity": entity,
            "score": score,
            "index": index,
            "start": start,
            "end": end,
        }
    )

File: examples/julia/tasks/token_classification/token_classification.jl
using Bytez

client = Bytez.init("YOUR BYTEZ KEY HERE")

model = client.model("dslim/bert-base-NER")

model.load()

input_text = "John Doe is a software engineer at Google."

result = model.run(input_text)

word_objects = result["output"]

for word_object in word_objects
	#  depending on the model, there may be additional props returned
	println(word_object)

	word = get(word_object, "word", "N/A")
	entity = get(word_object, "entity", "N/A")
	score = get(word_object, "score", 0.0)
	index = get(word_object, "index", 0)
	start = get(word_object, "start", 0)
	_end = get(word_object, "end", 0)

	println(
		Dict(
			"word" => word,
			"entity" => entity,
			"score" => score,
			"index" => index,
			"start" => start,
			"end" => _end,
		),
	)
end
